src/sources/file.rs

```diff
@@ -556,9 +556,19 @@ pub fn file_source(
             event
         });
         tokio::spawn(async move {
-            out.send_event_stream(&mut messages)
+            match out
+                .send_event_stream(&mut messages)
                 .instrument(span.or_current())
                 .await
+            {
+                Ok(()) => {
+                    info!("Finished sending.");
```

Comment:
This feels like it would be noisy - should this be debug or trace instead?

Reply:
Good question- I modeled this after other cases also using `info()`.

These sources are doing that: `file_descriptors`,  `http_scrape`,  `apache_metrics` , `socket` , `syslog` . Though socket and syslog in a slightly different behavior.

Reply:
I think there were some suggestions that we wanted "regular logs to tell things are running" but I'm not sure where things are there. @jszwedko what do you think?

Reply:
👍 this seems like more of a `debug` level log to me.

Reply:
Addressed in https://github.com/vectordotdev/vector/pull/14155/commits/0350b29c539af48a71f2a0b7d91568018e32f07c